### ISSUE TITLE
Deleted calendar from recurring event appears after editing instance

### ISSUE THREAD

**Issue by KolbeinNL** [owncloud/calendar#968]
### Steps to reproduce
1. Create event
2. Add time
3. Add recurring
4. Verify that the events are created
5. Remove instance
6. Change time or description
7. Verify that the removed instance reappeared

### Expected behaviour
The removed instance should be removed permanently

### Actual behaviour
The removed instance appeared again

### Server configuration
ubuntu 18.04

**Operating system**:
ubuntu

**Web server:**
Apache

**Database:**
postgres

**PHP version:**

**ownCloud version:** (see your admin page)
10.0.10

**Calendar version:** (see the apps page)
1.6.0

The networklog

![schermafdruk van 2019-02-03 19-54-10](https://user-images.githubusercontent.com/45788839/52181462-66b1ad80-27f2-11e9-99af-19cf4ed45b5a.png)

**Added the relevant PUT request as HAR**

[Archive 19-02-03 19-58-36.zip](https://github.com/owncloud/calendar/files/2825877/Archive.19-02-03.19-58-36.zip)

add a recurring event
![schermafdruk van 2019-02-03 19-50-56](https://user-images.githubusercontent.com/45788839/52181188-a88d2480-27ef-11e9-8890-8f38ef41f281.png)

delete instance
![schermafdruk van 2019-02-03 19-52-01](https://user-images.githubusercontent.com/45788839/52181220-f144dd80-27ef-11e9-9822-304fc835b912.png)

verify that the instance is deleted
![schermafdruk van 2019-02-03 19-52-15](https://user-images.githubusercontent.com/45788839/52181228-00c42680-27f0-11e9-9dd7-a3b6c0eb9e51.png)

change f.e. the subject
![schermafdruk van 2019-02-03 19-53-07](https://user-images.githubusercontent.com/45788839/52181230-0457ad80-27f0-11e9-976f-c4a86ddfc117.png)

observe the deleted event reoccurred
![schermafdruk van 2019-02-03 19-53-28](https://user-images.githubusercontent.com/45788839/52181232-07529e00-27f0-11e9-9641-f951e065a1ac.png)

### Client configuration

Firefox

**Operating system:**

Ubuntu

**CalDAV-clients:**

The browser app

data/owncloud.log
sensitive information

webserverlog
sensitive information

**Comment by Julian1998** [comment]
### Logs
#### Web server error log
```
Insert your webserver log here
```

#### Log file (data/owncloud.log)
```
Insert your ownCloud.log file here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```



Please fill that gaps ;) It would help ous a lot!

**Comment by KolbeinNL** [comment]
Hello Julian,

I cannot add webserver logs as it contains sensitive information such as email addresses. I have added lots of screenshots as well as the output of curl.. Is this enough?

**Comment by Julian1998** [comment]
@KolbeinNL Hi, The screenshots are good to understand the process of what's happening. But normally I still need infos about any errors ;) But in this situation the ss should be enough!

I' m gonna reproduce this bug when I find some time.

**Comment by karakayasemi** [comment]
This issue may resolved with https://github.com/owncloud/calendar/pull/964 . @KolbeinNL would you test this issue with master branch?

**Comment by karakayasemi** [comment]
This issue is resolved with #964. @KolbeinNL if you can reproduce this error with current master, do not hesitate to re-open this ticket.
